### PR TITLE
Fix performance discussion on opaque types

### PR DESCRIPTION
**Pull request by b-studios**
Abstract types are erased to their bound and thus this discussion of _performance_ overhead for them is wrong. Thanks @sjrd for pointing this out!

### CODE REVIEW

**Comment by b-studios** [comment]
@julienrf could you look at this PR? I am sadly no maintainer and can't mark you as reviewer ;)

File: _overviews/scala3-book/types-opaque-types.md [hunk 1]
```diff
@@ -98,8 +97,12 @@ Programming against the abstract module `Logarithms` can be very tedious and oft
 def someComputation(L: Logarithms)(init: L.Logarithm): L.Logarithm = ...
 ```
 
+#### Boxing Overhead
+Type abstractions, such as `type Logarithm` erase to their bound (which is `Any` in our case).
```
Comment (julienrf) [inline _overviews/scala3-book/types-opaque-types.md:101]: Could we add a link around “erase” to some page somewhere that describes what type erasure is?

**Review by julienrf — APPROVED** [review]
Thank you Jonathan! This looks good to me, I just have a small request.

File: _overviews/scala3-book/types-opaque-types.md (hunk 1 above)
Comment (b-studios) [inline _overviews/scala3-book/types-opaque-types.md:101]:
```suggestion
Type abstractions, such as `type Logarithm` [erase](https://www.scala-lang.org/files/archive/spec/2.13/03-types.html#type-erasure) to their bound (which is `Any` in our case).
```

**Pull request merged**